### PR TITLE
PMM-7659 Add external access to DB cluster

### PR DESCRIPTION
**Pull request by tiagomotasantos**
**What this PR does / why we need it**: 

- Adds `Switch` to DB Cluster creation page to enable external access for the DB cluster (exposes the cluster to the world)
- New `Switch` component was added that allows a tooltip, the idea is to contribute this Switch to core
- Found a bug in grafana: tooltips have lower z-index than the modal, which means tooltips don't show in the modal because they are hidden underneath. I will later open a PR on grafana to fix it on their side

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-7659

### CODE REVIEW

**Review by fabio-silva — APPROVED** [review]

File: public/app/percona/dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.test.tsx [hunk 1]
```diff
@@ -16,13 +16,16 @@ describe('DBClusterParameters::', () => {
     const memory = root.find('[data-qa="cluster-parameters-memory"]');
     const cpu = root.find('[data-qa="cluster-parameters-cpu"]');
     const disk = root.find('[data-qa="cluster-parameters-disk"]');
+    const expose = root.find('[data-qa="cluster-parameters-expose"]');
```
Comment (nicolalamacchia) [inline public/app/percona/dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.test.tsx:19]: Can we use the `dataQa` helper here from `@percona/platform-core`?

File: public/app/percona/dbaas/components/Switch/Switch.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React, { FC, useMemo } from 'react';
+import { Field } from 'react-final-form';
+import { cx } from 'emotion';
+import { Icon, Tooltip, Switch, useStyles } from '@grafana/ui';
+import { compose } from '@percona/platform-core/dist/shared/validators';
+import { SwitchFieldProps } from './Switch.types';
+import { getStyles } from './Switch.styles';
+import { SwitchFieldRenderProps } from './Switch.types';
+
+export const SwitchField: FC<SwitchFieldProps> = ({
```
Comment (nicolalamacchia) [inline public/app/percona/dbaas/components/Switch/Switch.tsx:10]: Why not adding this component to `platform-core`?

**Review by nicolalamacchia — APPROVED** [review]

File: public/app/percona/dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.test.tsx (hunk 1 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.test.tsx:19]: Done!

File: public/app/percona/dbaas/components/Switch/Switch.tsx (hunk 2 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/components/Switch/Switch.tsx:10]: Will add it after the upgrade.

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**